datamodel/src/main/scala/com/rasterfoundry/granary/package.scala

```diff
@@ -0,0 +1,14 @@
+package com.rasterfoundry.granary
+
+import io.estatico.newtype.macros.newtype
+import eu.timepit.refined._
+import eu.timepit.refined.api._
+import eu.timepit.refined.collection.Contains
+
+import java.util.UUID
+
+package object datamodel {
+  @newtype case class UserId(toUUID: UUID)
+  @newtype case class TokenId(toUUID: UUID)
+  @newtype case class Email(emailString: String Refined Contains[W.`'@'`.T])
```

Comment:
There's no license on it - but there is potentially a slightly better refined string for email [here](https://gist.github.com/gvolpe/cc7fc4589d1de631faaa5dcb48ef732e). I don't feel too strongly about it, but wanted to point you to it for future reference.

Reply:
I'll keep that in mind -- since we're responsible for creating tokens, I think it's probably fine for at least a bit for us to try to do a good job (especially since they're for us to remember who tokens are for at this point anyway), but if we run into problems we can enhance this.